service/main.ml

```diff
@@ -130,15 +135,29 @@ let submission_service =
     ~docv:"FILE"
     ["submission-service"]
 
-let cmd =
+let github_app_id =
+  Arg.required @@
+  Arg.opt Arg.(some string) None @@
+  Arg.info ["github-app-id"]
```

Comment:
There should be a Cmdliner version of this in current_gitlab

Reply:
I don't understand, but so far the pipeline doesn't use any gitlab repo.